src/System.Net.Http/tests/FunctionalTests/HttpCookieProtocolTests.cs

```diff
@@ -0,0 +1,413 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Test.Common;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public class HttpCookieProtocolTests : HttpClientTestBase
```

Comment:
There are already some cookie tests here:
https://github.com/dotnet/corefx/blob/ef0aca54454748002325b750c839dacba6ed3999/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs#L981-L1039
and here:
https://github.com/dotnet/corefx/blob/ef0aca54454748002325b750c839dacba6ed3999/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs#L1077-L1174
If we're creating a new test class for cookies, can we consolidate them to the same place?

Pre-existing your PR, but exacerbated every time we add tests to new places, our scheme for where we put tests in this suite is unwieldy.

Reply:
I will consolidate them into this file.